doc/use_cases.tex

```diff
@@ -13,7 +13,7 @@
           quantities of the same nature but expressed in different
           systems (sky coordinates in ICRS vs.~Gal). 
           This is partially supported by the actual VOTable schema (TIMESYS, COOSYS) but any change in these ad-hoc mapping 
-          elements or the addition of any other elements (e.g. X-ray energy band) would require VOTable version upgrades. 
+          elements or the addition of any other elements (e.g. definition of X-ray energy band) would require VOTable version upgrades. 
```

Comment:
I still don't much like this example; neither COOSYS nor TIMESYS even talk about X-rays or messengers, and where they need enumerated values (reference frames, timescales,...) they use vocabularies which are amendable without schema updates; that's what we have vocabularies for.

Reply:
Reference to X-ray EB dropped